src/app/pages/SearchResultsPage/GlobalSearchResultsView.tsx

```diff
@@ -30,6 +30,25 @@ export const GlobalSearchResultsView: FC<{
 
   const themes = getThemesForNetworks()
   const networkNames = getNetworkNames(t)
+
+  if (hasFixedNetwork()) {
```

Comment:
Isn't there a better way to filter search results? Does it makes queries to all the networks/layers still, in case of fixed network/paratime?

Reply:
Yes, good point. We shouldn't even make the queries. Will fix.

Reply:
So, looking at the code, we were already limiting the queries to the right network, but not the right layer. Now I have fixed that, too.

So we don't need to filter when displaying the results, so removed that. (But we still need custom code for displaying, because we don't even want to offer other networks, so that part remains.)